index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio Mobile Version Skeleton</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <header>
+        <div class="logo">
+            
+           <h4 class="logo">MY Logo</h4>
+            
+        </div>
+        <div class="navbar">
+            <img src="Union.svg" alt="Navbar-Image" class="nav-image">
+        </div>
+    </header>
+
+    <section class="headling">
+        <h2 class="intro">
+            I’m Mariane Glad to see you!
+        </h2>
+
+        <p class="professional-details">
+            I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+        </p>
+
+        <h4 class="social-media-intro">LET’S CONNECT</h4>
+
+        <img src="social-media-icons.svg" alt="" class="social-icon">
+
+        <img src="header-shapes-mobile.svg" alt="" class="header-shape-image">
+        <!-- <img src="header-shapes mobile@2x.svg" alt="" class="header-background"> -->
```

Comment:
- [ ] It's considered good practice to use background images as background images for containers and not in an `img` tag. Please make sure that the background image is used as a background image for the container of this section. You can use background image positioning and size properties to put the image in the right place.